### PR TITLE
Welcome view is almost done

### PR DESCRIPTION
**Pull request by muaathaw**
welcome view is almost done, we just need to fill in our words

### CODE REVIEW

File: src/views/WelcomeView.vue [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts"></script>
+
+<template>
+  <section class="section">
+    <div class="container">
+      <div class="content is-large">
+        <h1>Welcome to</h1>
+        <h1 class="title">ThreatMapper AI</h1>
+        <button class="button is-primary start-button">Press to start a conversation</button>
```
Comment (EmEhPa) [inline src/views/WelcomeView.vue:9]: Add 
```
            <RouterLink to="/chat">
              <button class="button is-primary custom-button">The text</button>
            </RouterLink>
```
and maybe change the text to just "Start chat"?

File: src/views/WelcomeView.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts"></script>
+
+<template>
+  <section class="section">
+    <div class="container">
+      <div class="content is-large">
```
Comment (EmEhPa) [inline src/views/WelcomeView.vue:6]: only content and not is-large

File: src/views/WelcomeView.vue [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<script setup lang="ts"></script>
+
+<template>
+  <section class="section">
+    <div class="container">
```
Comment (EmEhPa) [inline src/views/WelcomeView.vue:5]: Maybe add the box even if I am going to crate a component for it?

**Review by EmEhPa — CHANGES_REQUESTED** [review]

File: src/views/WelcomeView.vue (hunk 3 above)
Comment (muaathaw) [reply src/views/WelcomeView.vue:5]: fixed

File: src/views/WelcomeView.vue (hunk 2 above)
Comment (muaathaw) [reply src/views/WelcomeView.vue:6]: fixed as well

File: src/views/WelcomeView.vue (hunk 1 above)
Comment (muaathaw) [reply src/views/WelcomeView.vue:9]: Yes Boss!

File: src/views/WelcomeView.vue [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    </div>
+  </section>
+</template>
+
+<style scoped>
+.custom-button {
+  width: 250px;
+  margin: 0 80px;
+}
+.content {
```
Comment (EmEhPa) [inline src/views/WelcomeView.vue:24]: you should't need this

**Review by EmEhPa — CHANGES_REQUESTED** [review]

**Review by EmEhPa — APPROVED** [review]

**Pull request merged**